Automattic/woocommerce-payments
# Introduced deposit currency filter for transactions overview

Fixes #1341

#### Changes proposed in this Pull Request

- Introduced settlement currency filter on the transactions overview page

#### Testing instructions

- Prerequisite: a non-USD Stripe account should be used for the testing
- Ensure the client connected to a server running server#638 branch
- Clear account data cache in WCPay Dev
- Ensure deposits overview shows the proper currency
- Open transactions and ensure the filter is not available
- Change [this line](https://github.com/Automattic/woocommerce-payments-server/blob/update/1341-fixed-currency-filter/server/wp-content/rest-api-plugins/endpoints/wcpay/class-transactions-controller.php#L235) to `$summary['store_currencies'] = [ 'eur', 'usd', 'gbp' ];`
- Refresh the transactions page, ensure the filter appears and play around with the filter

#### Questions

- How to use a value other than 'all' in currency filter for default value?

-------------------
